Catappult/appcoins-wallet-android
# [eskills/uri-intent] - Implements startEskillsFlow via URI

**What does this PR do?**

Uses existing OneStepPaymentReceiver to receive URI which will be
BASE_HOST + /transaction/eskills. Note that no other activity can listen
to this path, since this activity listens to /transaction (perhaps in
the future we can change this to /inapp ?).
Sends this URI to SkillsActivity which will proceed with the normal
flow. This commit includes minor changes, like using price, currency and
sku from URI instead of having hardcoded ones.
Besides that, it also adds the new payment host to gradle.properties,
meaning that we are now listening for apichain.dev.catappult.io and
apichain-dev.blockchainds.com. This is only applicable to debug build
type, since on release type we were already listening to both of them

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] app/src/main/java/com/asfoundation/wallet/ui/OneStepPaymentReceiver.kt

**How should this be manually tested?**

Build and run 2048 from eskills/uri-intent. Choose AppCoins Wallet to deal with URI. Flow should proceed as expected.

**What are the relevant tickets?**

N/A

**Questions:**

N/A



**Code Review Checklist**
